lldb/bindings/interface/SBAddressRangeListExtensions.i

```diff
@@ -0,0 +1,13 @@
+%extend lldb::SBAddressRangeList {
```

Comment:
If we add a method to SBAddressRangeList:
```
  bool GetDescription(lldb::SBStream &description);
```
Then we can add this to this file:
```
STRING_EXTENSION_OUTSIDE(SBAddressRangeList)
```
And then we can print the SBAddressRangeList objects as user readable strings.